### PR TITLE
Established the link btw aws and github

### PR DESCRIPTION
**Pull request by sammy8825**

### CODE REVIEW

File: lib/hellouniverse.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const handler = async function (event: any) {
```
Comment (sarveshbhatnagar) [inline lib/hellouniverse.ts:1]: Please avoid function declarations like this, try to make it structured e.g. 

https://github.com/sarveshbhatnagar/userportal-backend/blob/main/index.js

Having said that, it is fine for now as these are for demonstration purposes.

File: lib/tenantconnect-backend-stack.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { Duration, Stack, StackProps } from 'aws-cdk-lib';
+import {Runtime, Function, Code} from 'aws-cdk-lib/aws-lambda';
+// import * as sns from 'aws-cdk-lib/aws-sns';
+// import * as subs from 'aws-cdk-lib/aws-sns-subscriptions';
+// import * as sqs from 'aws-cdk-lib/aws-sqs';
+import { Construct } from 'constructs';
+
+export class TenantconnectBackendStack extends Stack {
```
Comment (sarveshbhatnagar) [inline lib/tenantconnect-backend-stack.ts:8]: rename

File: template.yaml [hunk 3]
```diff
@@ -0,0 +1,175 @@
+Resources:
```
Comment (sarveshbhatnagar) [inline template.yaml:1]: Please add this file to ignored ones. I think this is the generated file. We don't want to commit this file to the branch.

File: test/tenantconnect-backend.test.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+test('SQS Queue and SNS Topic Created', () => {
+  const app = new cdk.App();
+  // WHEN
+  const stack = new TenantconnectBackend.TenantconnectBackendStack(app, 'MyTestStack');
+  // THEN
+  
+  const template = Template.fromStack(stack);
+
+  template.hasResourceProperties('AWS::SQS::Queue', {
```
Comment (sarveshbhatnagar) [inline test/tenantconnect-backend.test.ts:13]: Did you create SQS Queue?

**Review by sarveshbhatnagar — COMMENTED** [review]
please see the comments

File: lib/tenantconnect-backend-stack.ts (hunk 2 above)
Comment (sammy8825) [reply lib/tenantconnect-backend-stack.ts:8]: I have changed the name in the following pull request

File: test/tenantconnect-backend.test.ts (hunk 4 above)
Comment (sammy8825) [reply test/tenantconnect-backend.test.ts:13]: Removed the same as we just use lambda function

File: bin/tenantconnect-backend.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+import * as cdk from 'aws-cdk-lib';
+import { TenantconnectBackendStack } from '../lib/tenantconnect-backend-stack';
+
+const app = new cdk.App();
+new TenantconnectBackendStack(app, 'TenantconnectBackendStack');
```
[the remark this answers is not in the document]
Comment (sammy8825) [reply bin/tenantconnect-backend.ts:6]: I have corrected the naming in the following pr

**Comment by sarveshbhatnagar** [comment]
Not working on this anymore, instead focusing on code structure and move from there.

**Pull request closed without merging**